prodtests/full-system-test/run-workflow-on-inputlist.sh

```diff
@@ -70,13 +70,7 @@ else
   exit 1
 fi
 
-if [ -f dpl-workflow.sh ]; then
-    echo "Use dpl-workflow.sh macro passed as input"
-else
-    echo "Use dpl-workflow.sh macro from ${MYDIR}"
-    cp $O2_ROOT/prodtests/full-system-test/dpl-workflow.sh .
-fi
-start_process ./dpl-workflow.sh
+start_process ${DPL_WORKFLOW_FROM_OUTSIDE:-$MYDIR/dpl-workflow.sh}
```

Comment:
Ok, for running on the grid it means one more setting, but if you like this better, no problem with me.